Configure.ps1

```diff
@@ -146,10 +146,10 @@ of the visual studio solution (App.Config and Web.Config) so that they are consi
 #Clean-Ups the applications and serice principal in Azure AD.
```

Comment:
Two typos ("clean-ups" and "serice"): "Remove the applications and service principals registered for the sample", or "Remove the application registrations previously created for the sample"